types/markdown-it-link-attributes/index.d.ts

```diff
@@ -1,18 +1,26 @@
 // Type definitions for markdown-it-link-attributes 3.0
 // Project: https://github.com/crookedneighbor/markdown-it-link-attributes
 // Definitions by: Katy Richard <https://github.com/cowpewter>
+//                 Piotr Błażejewicz <https://github.com/cowpewter>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import MarkdownIt = require('markdown-it');
+import { PluginWithOptions } from "markdown-it";
+import { RenderRule } from "markdown-it/lib/renderer";
 
 declare namespace markdownItLinkAttributes {
+    // const defaultRender: RenderRule;
```

Comment:
Maybe just delete this commented out line?

Reply:
amended, thx!